numba/unsafe/ndarray.py

```diff
@@ -49,6 +50,8 @@ def to_fixed_tuple(typingctx, array, length):
     - No boundchecking.
       If *length* is longer than *array.size*, the behavior is undefined.
     """
+    # QUESTION: Should this intrinsic be removed, gracefully deprecated or left
```

Comment:
I'm not sure it adds any value?

@sklam do you recall how prevalent this is in external code?

Reply:
This might not be representative but I found [two projects on github](https://github.com/search?utf8=%E2%9C%93&q=%22to_fixed_tuple%22+-filename%3A%22test_unsafe_intrinsics.py%22+-filename%3A%22ndarray.py%22+-filename%3A%22devicearray.py%22+extension%3Apy&type=Code&ref=advsearch&l=&l=) that use this function in four places.

Reply:
Just to clarify, this is the second intrinsic that will be removed?

Reply:
@esc Yes, this is the second intrinsic.

Reply:
✅

Reply:
I am going to go out on a limb and say that in light of the limited external usage, it is probably reasonable to delete this in the interest of code cleanliness. I am happy to give the two projects using this on github a heads-up if you agree. I have deleted it for now. Please yell if you want it restored.